### PR TITLE
Improvements to the CFEngine glossary

### PR DESCRIPTION
**Pull request by olehermanse**
- Glossary: Added some details on classes and class guards
- Glossary: Reformatted markdown for nicer editing / review
- Glossary: Fixed capitalization, grammar, markup

### CODE REVIEW

File: overview/glossary.markdown [hunk 1]
```diff
@@ -28,7 +28,20 @@ A promise body is the description of exactly what is promised (as opposed to wha
...
 #### Call Collect
-#### Classes
+#### Class
+
+Classes are used to classify a system (or the state of it) and to make decisions in CFEngine policy.
+Classes are sometimes referred to as contexts.
+
+#### Class expressions
+
+Multiple classes separated by operators (and, or) to make more complex decisions.
```
Comment (larsewi) [inline overview/glossary.markdown:37]: (and, or, not)

**Review by larsewi — APPROVED** [review]

File: overview/glossary.markdown [hunk 2]
```diff
@@ -8,58 +8,76 @@ tags: [guide, glossary]
...
 #### Enterprise API
 #### Enterprise reporting
-#### File Structure
+#### File structure
 #### Frequency
 #### Functions
 #### Host
 
-UNIX terminology for a computer the runs "guest programs." In practice, "host" is a synonym for "computer."
+UNIX terminology for a computer the runs "guest programs."
```
Comment (craigcomstock) [inline overview/glossary.markdown:73]:
```suggestion
UNIX terminology for a computer that runs "guest programs."
```

**Review by craigcomstock — COMMENTED** [review]
cool, one existing typo, otherwise :+1:

**Comment by olehermanse** [comment]
@cf-bottom please build this one in Jenkins now :)

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/362/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/362/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-362/output/_site/

**Review by craigcomstock — APPROVED** [review]

**Comment by olehermanse** [comment]
Backport: https://github.com/cfengine/documentation/pull/3064

**Pull request merged**